src/Tribe/RSVP.php

```diff
@@ -323,6 +323,7 @@ public function render_rsvp_step( $ticket_id, $step = null ) {
 			'must_login' => ! is_user_logged_in() && $this->login_required(),
 			'login_url'  => self::get_login_url( $post_id ),
 			'threshold'  => $blocks_rsvp->get_threshold( $post_id ),
+			'going'      => tribe_get_request_var( 'going', '' ),
```

Comment:
This looks good. Make sure you add a new `@since` tag within the `render_rsvp_step` docblock.

Reply:
Thanks @redscar